pkg/client/restclient/config.go

```diff
@@ -87,6 +88,9 @@ type Config struct {
 
 	// Maximum burst for throttle
 	Burst int
+
+	// Rate limiter for limiting connections to the master from this client. If present overwrites QPS/Burst
```

Comment:
Nothing hard - I'm just following the 'add, don't remove' principle in case it was used somewhere in a serialized form.